# service file name 'rg_check_target_srv'


# actual position
geometry_msgs/Point x

# target position
geometry_msgs/Point xt

---

# the twist to send to 'cmd_vel'
geometry_msgs/Twist vel
